manifests/repo.pp

```diff
@@ -19,6 +20,17 @@
     notify  => Anchor['pulpcore::repo'],
   }
 
+  if $facts['os']['release']['major'] == '8' {
```

Comment:
```suggestion
  if $facts['os']['release']['major'] != '7' {
```

to match tests? (and the idea of future proofing) :)

Reply:
I copied it from here: https://github.com/theforeman/puppet-foreman/blob/9e483426c511e7a7a2648c2eaba18f50ac7df8f9/manifests/repo.pp#L44
But you're right.